packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -43,3 +44,125 @@ export function pretty(arg: unknown): number | string {
 	}
 	return JSON.stringify(arg);
 }
+
+const schemaFactory = new SchemaFactory("test");
+
+export class DeepTreeNode extends schemaFactory.objectRecursive("deep-tree-node", {
```

Comment:
We should document this type and its properties.

Reply:
Updated